src/Functions/Public/composition-service/Add-vRACustomForm.ps1

```diff
@@ -0,0 +1,94 @@
+function Add-vRACustomForm {
+<#
+    .SYNOPSIS
+    Add a vRA Custom Form for a Blueprint
+
+    .DESCRIPTION
+    Add a vRA Custom Form for a Blueprint
+
+    .PARAMETER BlueprintId
+    Specify the ID of a Blueprint
+
+    .PARAMETER Body
+    The JSON string containing the custom form
+
+    .INPUTS
+    System.String
+
+    .OUTPUTS
+    System.String
+
+    .EXAMPLE
+    $JSON = Get-Content -Path ~/CentOS.json -Raw
+    Add-vRACustomForm -BlueprintId "CentOS" -Body $JSON
+
+    .EXAMPLE
+    $JSON = Get-Content -Path ~/CentOS.json -Raw
+    Get-vRABlueprint -Name "CentOS" | Add-vRACustomForm -Body $JSON
+
+    .EXAMPLE
+    $Form = Get-vRABlueprint -Name "CentOS" | Get-vRACustomForm
+    Add-vRACustomForm -BlueprintId "RHEL7" | Add-vRACustomForm -Body $Form.JSON
+
+
+#>
+[OutputType('System.Management.Automation.PSObject')]
```

Comment:
Add SupportsShouldProcess with ConfirmImpact set to low
```SupportsShouldProcess,ConfirmImpact="Low"```

That way it'll be possible to test this command with whatif

Reply:
fixed